src/background/init-context-menus.ts

```diff
@@ -0,0 +1,15 @@
+function openNewTabWithWallet() {
+  return chrome.tabs.create({ url: 'full-page.html' });
+}
+
+export function initContextMenuActions() {
+  chrome.contextMenus.removeAll();
+
+  chrome.contextMenus.create({
+    title: 'Open Hiro Wallet full page',
```

Comment:
I wonder if this should be something else, like `Open in full page mode`. cc @jasperjansz for ideas

Reply:
Yep down for phrasing alternatives cc/ Mark often-has-opinions-on-copy Hendrickson @markmhx

Reply:
...in a new tab might also be better terminology

Reply:
@kyranjamie how about `Open Hiro wallet in new tab` and merge it?